### PR TITLE
The grammar mistakes were corrected

### PR DESCRIPTION
**Pull request by ADShenoi**

### CODE REVIEW

**Comment by StijnSimons** [comment]
How to get contributor

File: README.md [hunk 1]
```diff
@@ -31,14 +31,14 @@ A Minecraft pathfinder bot.
...
+The easiest way to install Baritone is to install [Impact](https://impactclient.net/), which comes with Baritone. The second easiest way (for 1.12.2 only) is to install the v1.2.* forge API jar from [releases](https://github.com/cabaletta/baritone/releases). Otherwise, see [Installation & setup](SETUP.md). Once Baritone is installed, look [here](USAGE.md) for instructions on how to use it.
 
 For 1.14.4, [click here](https://www.dropbox.com/s/rkml3hjokd3qv0m/1.14.4-Baritone.zip?dl=1). Or [with optifine](https://github.com/cabaletta/baritone/issues/797).
 
 This project is an updated version of [MineBot](https://github.com/leijurv/MineBot/),
 the original version of the bot for Minecraft 1.8.9, rebuilt for 1.12.2 and 1.13.2. Baritone focuses on reliability and particularly performance (it's over [30x faster](https://github.com/cabaletta/baritone/pull/180#issuecomment-423822928) than MineBot at calculating paths).
 
-Have committed at least once a day from Aug 1 2018 to Aug 1 2019.
+Have committed at least once a day from Aug 1, 2018, to Aug 1, 2019.
```
Comment (ZeroMemes) [inline README.md]: Unnecessary comma after 2018

Comment (ZeroMemes) [reply README.md]: Wait nevermind lol!!!

**Comment by ZeroMemes** [comment]
>good, but I don't think that api necessarily has to be capitalized. But they both work.

API is an abbreviation

**Pull request merged**